<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErodeDialog</class>
 <widget class="QDialog" name="ErodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="erodeLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>40</y>
     <width>181</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Display Manner</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="displayLabel">
           <property name="text">
            <string>DisPlay 
Manner</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioOpenCV">
             <property name="text">
              <string>Show in OpenCV</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioQt">
             <property name="text">
              <string>Show in Qt</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="shapeLabel">
             <property name="text">
              <string> Kernel 
 Shape</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="radioRect">
               <property name="text">
                <string>RECT</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioEllipse">
               <property name="text">
                <string>Ellipse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioCross">
               <property name="text">
                <string>CROSS</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="sizeLabel">
             <property name="text">
              <string>Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="excuteButton">
           <property name="text">
            <string>Excute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
